.github/workflows/CreateRelease.ps1

```diff
@@ -72,25 +72,25 @@ $currentRelease = $null
     # All releases that are not the latest release are rejected.
     if(-Not($currentRelease -eq $lastRelease)) {
         $updateStatusObj = @{
-            status="rejected"
-            comment="Rejected by automation (reason: not the latest)"
-        }
+            "status" = "rejected"
```

Comment:
I think the quotes for the keys are optional. I tested it out and it worked with and without them